tests/dialects/test_snowflake.py

```diff
@@ -1253,6 +1253,24 @@ def test_snowflake(self):
                 "duckdb": "SELECT ((1) OR (0))",
             },
         )
+        self.validate_all(
```

Comment:
We don't need this one, there's already a test that suffices in `test_presto.py`.